File: src/Omnia.ProcessManagement.Web/client/fx/models/data/shapedefinitions/DrawingFreeformShapeDefinition.ts
import { FabricShapeData } from '../../../processshape/fabricshape/FabricShapeData';
import { DrawingShapeDefinition } from './DrawingShapeDefinition';

export interface DrawingFreeformShapeDefinition extends DrawingShapeDefinition {
    nodes: FabricShapeData[];
}

File: src/Omnia.ProcessManagement.Models/ShapeDefinitions/DrawingFreeformShapeDefinition.cs
using Omnia.ProcessManagement.Models.FabricShape;
using Omnia.ProcessManagement.Models.Shapes;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ShapeDefinitions
{
    public class DrawingFreeformShapeDefinition : DrawingShapeDefinition
    {
        public List<FabricShapeData> Nodes { get; set; }
    }
}
